vita3k/shader/src/translator/alu.cpp

```diff
@@ -1416,7 +1412,7 @@ bool USSETranslatorVisitor::vdual(
     std::string disasm_str = fmt::format("{:016x}: ", m_instr);
 
     auto do_dual_op = [&](Opcode code, std::vector<Operand> &ops, Operand &dest,
-        Imm4 write_mask_dest, const DualOpInfo &code_info) {
+                          Imm4 write_mask_dest, const DualOpInfo &code_info) {
```

Comment:
all the other format stuff could probably do in its own commit if you can

thank you for cleaning up my mess 🙂